cep-recipe-jinja.md

```diff
@@ -0,0 +1,125 @@
+# Jinja functions in recipes
+
+The new recipe format has some Jinja functionalities. We want to specify what functions exist and their expected behavior.
```

Comment:
I feel this is lacking a short intro. At least we should link into the previous CEPs.

```suggestion
The new recipe format (introduced by CEP-XX, CEP-XY) has some Jinja functionalities. We want to specify what functions exist and their expected behavior.
```